core/src/test/java/org/elasticsearch/index/replication/ESIndexLevelReplicationTestCase.java

```diff
@@ -254,7 +255,7 @@ private IndexShard newShard(boolean primary, DiscoveryNode node, IndexMetaData i
         public int indexDocs(final int numOfDoc) throws Exception {
             for (int doc = 0; doc < numOfDoc; doc++) {
                 final IndexRequest indexRequest = new IndexRequest(index.getName(), "type", Integer.toString(docId.incrementAndGet()))
-                    .source("{}");
+                    .source("{}").waitForActiveShards(ActiveShardCount.NONE);
```

Comment:
I have no idea why that is there actually, I don't recall and it looks like remnants from some experimentation. I'm removing it.